packages/ui/types/ChainMetaData.ts

```diff
@@ -38,3 +38,7 @@ export const supportedChainIdToConfig: {
   [chapel.chainId]: { enabled: true, supported: config.isDevelopment || config.isTestnetEnabled },
   [ganache.chainId]: { enabled: config.isDevelopment, supported: config.isDevelopment },
 };
+
+export interface CrossFusePools {
```

Comment:
```suggestion
export interface FusePoolsPerChain {
```

I'm a fan of this `Cross` naming. It think it's a bit misleading. The functionalty is not really "across" chains it's just showing data from multiple chains at once.

In my opinion `FusePoolsPerChain` is a bit better describing what the type is.